neurom/view/view.py

```diff
@@ -95,6 +124,7 @@ def plot_tree(ax, tree, plane='xy',
         linewidth(float): all segments are plotted with this width, but only if diameter_scale=None
         color(str or None): Color of plotted values, None corresponds to default choice
         alpha(float): Transparency of plotted values
+        scale_with_axis(bool): scale linewidths with axis data coordinates
```

Comment:
At the level of this public function, I would prefer a simpler argument name that appeals to anyone. Maybe `realistic_diameters` ?